dbclient/metrics-jdbc/src/main/java/io/helidon/dbclient/metrics/jdbc/JdbcMetricsExtensionProvider.java

```diff
@@ -16,20 +16,22 @@
 package io.helidon.dbclient.metrics.jdbc;
 
 import io.helidon.common.config.Config;
-import io.helidon.dbclient.jdbc.HikariCpExtension;
-import io.helidon.dbclient.jdbc.spi.HikariCpExtensionProvider;
+import io.helidon.dbclient.jdbc.JdbcCpExtension;
+import io.helidon.dbclient.jdbc.spi.JdbcCpExtensionProvider;
 
 /**
- * {@link HikariCpExtensionProvider} implementation for {@link HikariMetricsExtension}.
+ * {@link io.helidon.dbclient.jdbc.spi.JdbcCpExtensionProvider} implementation for {@link JdbcMetricsExtension}.
  */
-public class JdbcMetricsExtensionProvider implements HikariCpExtensionProvider {
+public class JdbcMetricsExtensionProvider implements JdbcCpExtensionProvider {
```

Comment:
Please check with @tjquinno  if this is aligned with our plans for Metrics API in Helidon

Reply:
I moved this SPI to Hikari module. Related metrics module is Hikari specific so there is no need to keep common JDBC SPI for this purpose.
Now Hikari module just finds it's metrics if they are on CP./MP.